EVENT_PREP.md

```diff
@@ -38,3 +38,7 @@ These two posts need to include:
 Need to set up chairs and TV in venue. 
  
 Currently the venue sponsor takes care of the drinks fridge being stocked. And the pizza is ordered on the night, so there isn't any prep currently.
+
+## Dat after the event
```

Comment:
Should this read "Day after the event"?

Reply:
Certainly should. I've updated that commit.